service/service.go

```diff
@@ -75,22 +78,27 @@ func Create(bindAddr, vaultPath, bucketName, serviceAuthToken, downloadServiceTo
 		IsPublishing:  isPublishing,
 	}
 
+	// Create healthcheck object with versionInfo
+	versionInfo, err := healthcheck.NewVersionInfo(buildTime, gitCommit, version)
+	if err != nil {
+		log.Event(ctx, "initialising healthcheck without versionInfo", log.Error(err))
+	}
+	hc := healthcheck.New(versionInfo, healthCheckRecovery, healthCheckInterval)
```

Comment:
Instead of `healthCheckRecovery`, `healthCheckCriticalTimeout`

Reply:
I see that you did the following in dp-frontend-filter-dataset-controller:
```
	Interval             string `envconfig:"HEALTHCHECK_INTERVAL"`
	CriticalTimeout      string `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
```
I will do the same, but maybe HealthcheckInterval and HealthcheckCtiricalTimeou

Reply:
I changed the name in the other apps so it is more explicit as there maybe other critical timeouts or intervals. My naming suggestion above should still stand